# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/android/AccessibleWrap.cpp
# Commit: 4fab2284cd43
# Full Hash: 4fab2284cd43fd944bdab7ca87df9a3c2c6ea9ad
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 3: Directly handle scrolling start events on Android instead of converting them to virtual cursor change events. r=eeejay
#   
#   Depends on D192643
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192644
# ==============================================================================

diff -r 16707adce3d7 -r 4fab2284cd43 accessible/android/AccessibleWrap.cpp
--- a/accessible/android/AccessibleWrap.cpp	Wed Nov 29 03:53:19 2023 +0000
+++ b/accessible/android/AccessibleWrap.cpp	Wed Nov 29 03:53:19 2023 +0000
@@ -61,19 +61,6 @@
 nsresult AccessibleWrap::HandleAccEvent(AccEvent* aEvent) {
   auto accessible = static_cast<AccessibleWrap*>(aEvent->GetAccessible());
   NS_ENSURE_TRUE(accessible, NS_ERROR_FAILURE);
-  DocAccessibleWrap* doc =
-      static_cast<DocAccessibleWrap*>(accessible->Document());
-  if (doc) {
-    switch (aEvent->GetEventType()) {
-      case nsIAccessibleEvent::EVENT_SCROLLING_START: {
-        accessible->PivotTo(
-            java::SessionAccessibility::HTML_GRANULARITY_DEFAULT, true, true);
-        break;
-      }
-      default:
-        break;
-    }
-  }
 
   nsresult rv = LocalAccessible::HandleAccEvent(aEvent);
   NS_ENSURE_SUCCESS(rv, rv);